### DISCUSSION TITLE
View tif images

### DISCUSSION THREAD

**Discussion by maaikez** [filebrowser/filebrowser#1174] [Ideas]
**Is your feature request related to a problem? Please describe.**
I want to view tif images, but that is not possible in a browser. The created thumbnails show an image, but when clicking on the thumbnail to show the fill sized image tif, it is not possible to view the image in the browser (Firefox and Mozilla at least).

**Describe the solution you'd like**
A way to show the (full size) tif images. For example by converting them to another format (jpg?) or doing some other magic. Can be a setting whether to convert them or not (since that might take some time?). 

**Describe alternatives you've considered**
I tried to add a plugin to the browser, but that does not work. And since I have Linux, I have no Safari or Internet Explorer / Edge browser.

**Comment by maaikez** [comment]
I found https://conceptclarity.ca/display-tiff-files-web/ for example (which mentions to use https://github.com/photopea/UTIF.js/). Or https://medium.com/@thad.humphries/displaying-tiff-with-tiff-js-and-html5-canvas-dd02fe9f1a5d, which uses https://github.com/seikichi/tiff.js (I did not look in great detail into this, but it might help).

**Comment by WeidiDeng** [comment]
Fixed in [pr](https://github.com/filebrowser/filebrowser/pull/1222).

**Reply by maaikez** [reply]
Great, thanks a lot! Will test it later.
